package.json

```diff
@@ -69,15 +69,12 @@
 		"build-css:reader-mobile": "node-sass --importer node_modules/node-sass-package-importer/dist/cli.js --include-path client client/assets/stylesheets/reader-mobile.scss public/reader-mobile.css --output-style compressed && yarn run -s postcss public/reader-mobile.css",
 		"build-devdocs:search-index": "node bin/generate-devdocs-search-index.js",
 		"build-devdocs:selectors-index": "node bin/generate-devdocs-selectors-index.js",
-		"build-desktop": "run-s build-desktop:source build-desktop:install-app-deps build-desktop:app",
-		"build-desktop:source": "run-s build-desktop:config build-desktop:server build-desktop:client build-desktop:static",
-		"build-desktop:config": "node desktop/bin/build-desktop-config.js",
-		"build-desktop:secrets": "node desktop/bin/build-desktop-secrets.js",
-		"build-desktop:server": "BROWSERSLIST_ENV=server CALYPSO_ENV=desktop webpack --config client/webpack.config.desktop.js --stats-preset errors-only $WEBPACK_OPTIONS",
-		"build-desktop:client": "CALYPSO_ENV=desktop yarn run build-client-evergreen",
-		"build-desktop:static": "npx --no-install ncp static desktop/public",
-		"build-desktop:install-app-deps": "cd desktop && yarn install --frozen-lockfile",
+		"build-desktop": "cd desktop && yarn install --frozen-lockfile && yarn run build",
+		"build-desktop:source": "node -e \"console.log( 'Deprecated (legacy). Remove on TeamCity configuration update.')\"",
```

Comment:
I love the deprecation messages!!! Great UX

Reply:
That was all Nich!